gridcomps/History/Sampler/MAPL_TrajectoryMod_smod.F90

```diff
@@ -554,12 +556,21 @@
          integer :: nx2
          logical :: EX ! file
          logical :: zero_obs
+         integer, allocatable :: sendcount(:), displs(:)
+         integer :: recvcount
+         integer :: is, ie, ierr
+         integer :: M, N, ip
+
+
+         real(kind=REAL64), allocatable :: lons_chunk(:)
+         real(kind=REAL64), allocatable :: lats_chunk(:)
+         real(kind=REAL64), allocatable :: times_R8_chunk(:)
```

Comment:
Unless there is also an R4 version, then I'd prefer not to put KIND in the name of variables.

Reply:
OK. deleted.